### PR TITLE
AUI-2038

### PR DESCRIPTION
**Pull request by blzaugg**

### CODE REVIEW

**Comment by blzaugg** [comment]
AUI-2037 https://github.com/thiago-rocha/alloy-ui/pull/39

**Comment by blzaugg** [comment]
master-deprecated: https://github.com/thiago-rocha/alloy-ui/pull/49

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by thiago-rocha** [comment]
Pull request submitted to https://github.com/mairatma/alloy-ui/pull/288. [See changes here.](https://github.com/thiago-rocha/alloy-ui/compare/257b4e6~1...257b4e6) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-datatable/js/aui-datatable-base-options-cell-editor.js [hunk 1]
```diff
@@ -233,11 +331,9 @@ BaseOptionsCellEditor = A.Component.create({
...
                 instance.set('options', options);
 
-                instance._uiSetValue(
-                    instance.get('value')
-                );
-
-                instance.toggleEdit();
+                if (instance.get('hideEditContainerOnSave')) {
+                    instance.toggleEdit();
+                }
```
Comment (blzaugg) [inline src/aui-datatable/js/aui-datatable-base-options-cell-editor.js]: Related: https://github.com/blzaugg/alloy-ui/commit/9d33c634e45e29ab1c336d80093becc53c95c6cf

**Pull request closed without merging**